static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -1084,6 +1084,10 @@ function WidgetViewerModal(props: Props) {
                               });
                             }}
                             disabled={!hasEditAccess}
+                            title={
+                              !hasEditAccess &&
+                              t('You do not have permission to edit this dashboard')
```

Comment:
```suggestion
                              t('You do not have permission to edit this widget')
```
It makes more sense in my head but feel free to ignore